cypress/support/pages/contactForm.pageObject.js

```diff
@@ -4,11 +4,11 @@ class ContactFormPageObject extends PageObject {
   url = '/index.html';
 
   get emailField() {
-    return cy.get('#recipient-email');
+    return cy.get('#email');
   }
 
   get nameField() {
-    return cy.get('#recipient-name');
+    return cy.get('#name');
```

Comment:
Why do we need to change these values?

Reply:
Indeed, we don't need to change value for email, but we have error for name:

Timed out retrying after 4000ms: cy.type() failed because this element is not visible:
<input type="text" class="form-control" id="recipient-name">
This element <input#recipient-name.form-control> is not visible because its parent <div#exampleModal.modal.fade> has CSS property: display: none
Fix this problem, or use {force: true} to disable error checking